client/my-sites/plans-features-main/components/plan-upsell-modal/free-plan-free-domain-dialog.tsx

```diff
@@ -85,185 +92,118 @@ export function FreePlanFreeDomainDialog( {
-										/>
-									),
-								},
-							}
-						) }
-					</TextBox>
-				</ListItem>
-				<ListItem>
-					<div>
-						<CrossIcon icon="cross" size={ 24 } />
-					</div>
-					<TextBox>
-						{ translate(
-							'{{strong}}No ad-free experience:{{/strong}} Your visitors will see external ads on your site.',
-							{
-								components: { strong: <strong></strong> },
-							}
-						) }
-					</TextBox>
-				</ListItem>
-				<ListItem>
-					<div>
-						<CrossIcon icon="cross" size={ 24 } />
-					</div>
-					<TextBox>
-						{ translate(
-							'{{strong}}No unlimited professional customer support{{/strong}} (only community forums)',
-							{
-								components: { strong: <strong></strong> },
-							}
-						) }
-					</TextBox>
-				</ListItem>
-				<ListItem>
-					<div>
-						<CrossIcon icon="cross" size={ 24 } />
-					</div>
-					<TextBox noBottomGap>
-						{ translate(
-							'{{strong}}No extra storage:{{/strong}} You only get 1GB for photos, videos, media, and documents.',
-							{
-								components: { strong: <strong></strong> },
-							}
-						) }
-					</TextBox>
-				</ListItem>
+				{ featureUpsells.map( ( upsellItem ) => (
+					<ListItem>
+						<div>
+							<CrossIcon icon="cross" size={ 24 } />
+						</div>
+						<TextBox bold>{ upsellItem }</TextBox>
+					</ListItem>
+				) ) }
 			</List>
-			<TextBox>
-				{ planTitle &&
-					translate(
-						'Unlock {{strong}}all of{{/strong}} these features with a %(planTitle)s plan, starting at just %(planPrice)s/month, {{break}}{{/break}} with a 14-day money back guarantee.',
-						{
-							args: {
-								planTitle,
-								planPrice: formatCurrency( planPriceMonthly, currencyCode, {
-									stripZeros: true,
-								} ),
-							},
-							components: { break: <br />, strong: <strong></strong> },
-						}
-					) }
+			<TextBox marginBottom={ 8 }>
+				{ translate(
+					'Unlock all of these features with a %(planTitle)s plan, starting at just %(planPrice)s/month.',
+					{
+						args: {
+							planTitle: basicPlanUpsellInfo.title,
+							planPrice: basicPlanUpsellInfo.formattedPriceMonthly,
+						},
+					}
+				) }
 			</TextBox>
 			<TextBox>
-				{ domainProductCost &&
-					translate(
-						'As a bonus, you will get a custom domain - like {{strong}}{{italic}}yourgroovydomain.com{{/italic}}{{/strong}} - {{break}}{{/break}} free for the first year (%(domainPrice)s value).',
-						{
-							args: {
-								domainPrice: formatCurrency( domainProductCost, currencyCode, {
-									stripZeros: true,
-								} ),
-							},
-							components: {
-								strong: <strong></strong>,
-								italic: <i></i>,
-								break: <br />,
-							},
-						}
-					) }
+				{ translate(
+					'{{strong}}Need premium themes, live chat support, and advanced design tools?{{/strong}}{{break}}{{/break}}Go with our %(planTitle)s plan, starting at just %(planPrice)s/month. All annual plans come with a 14-day money-back guarantee.',
```

Comment:
Note that the original copy says "All plans come with a 14-day ...", which is not accurate since it's 7 days for a monthly plan.